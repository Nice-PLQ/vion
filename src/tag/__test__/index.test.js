import { mount } from '@vue/test-utils';
import Tag from '../tag.vue';

describe('Tag', () => {
  test('渲染', async () => {
    const wrapper = mount(Tag, {
      slots: {
        default: '标签'
      },
      props: {
        backgroundColor: '#07C160',
        textColor: '#000',
      }
    });

    expect(wrapper.element).toMatchSnapshot();
    expect(wrapper.text()).toBe('标签');

    await wrapper.trigger('click');
    expect(wrapper.emitted('click')).toBeTruthy();
  });
});
