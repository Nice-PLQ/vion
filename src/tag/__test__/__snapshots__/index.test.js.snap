// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tag 渲染 1`] = `
<div
  backgroundcolor="#07C160"
  class="v-tag v-tag_medium"
  style="color: rgb(0, 0, 0);"
>
  
  标签
  
</div>
`;
